noxfile.py

```diff
@@ -97,17 +97,19 @@ def run_pytest(
     opts = opts or {}
     pytest_opts = []
     pytest_env = {
-        "USERNAME": session.env.get("USERNAME"),
-        "PATH": session.env.get("PATH"),
-        "USERPROFILE": session.env.get("USERPROFILE"),
+        "USERNAME": session.env.get("USERNAME") or os.environ.get("USERNAME"),
+        "PATH": session.env.get("PATH") or os.environ.get("PATH"),
+        "USERPROFILE": session.env.get("USERPROFILE") or os.environ.get("USERPROFILE"),
```

Comment:
This change is from a different PR, right? I'm not sure about it, I will comment on it there.

Reply:
yes, I just wanted the ci to be green to make sure something else wasn't broken, will update ofc